packages/ui5-config/src/ui5config.ts

```diff
@@ -154,6 +154,14 @@ export class UI5Config {
         ui5Libraries: string[],
         ui5Theme = 'sap_fiori_3'
     ): UI5Config {
+        // Libraries to always load
+        const defaultLibraries = ['sap.m', 'sap.ushell', 'sap.ui.core'];
```

Comment:
Not ideal to have that here. It makes sense for the Fiori tools use cases because we always have an flp preview and need `sap.ushell` but this is a generic module to modify ui5.yaml files and not all consumers might want these libs in there. So, please remove that here, and add the libs to the already existing `ui5Libraries` parameter when you call the function